src/app/store/actions/VehicleTestResultModel.actions.spec.ts

```diff
@@ -1,43 +1,16 @@
 import {
   EVehicleTestResultModelActions, GetVehicleTestResultModel, GetVehicleTestResultModelSuccess, GetVehicleTestResultModelFailure
 } from './VehicleTestResultModel.actions';
-import { TestResultModel } from '@app/models/test-result.model';
+import {TestResultModel} from '@app/models/test-result.model';
 
-const testResultModelExample: TestResultModel = {
-  testerStaffId: '21',
-  testStartTimestamp: new Date(),
-  odometerReadingUnits: 'test',
-  testEndTimestamp: new Date(),
-  testStatus: 'new',
-  testTypes: [],
-  vehicleClass: null,
-  testStationName: 'test',
-  vehicleId: '213',
-  noOfAxles: 2,
-  vehicleType: 'none',
-  countryOfRegistration: 'Ro',
-  preparerId: '231',
-  odometerReading: 123,
-  vehicleConfiguration: 'none',
-  testStationType: 'none',
-  reasonForCancellation: 'none',
-  testerName: '213',
-  vrm: '123',
-  testStationPNumber: '123',
-  numberOfSeats: 123,
-  testerEmailAddress: '123@123.com',
-  euVehicleCategory: '123',
-  vin: '213',
-  vehicleSize: '213',
-  preparerName: '213'
-}
+const testResultModelExample = {} as TestResultModel;
```

Comment:
Well done. Howver, I would include 2 -5 keys to show relevant intention instead of only {}